node/node_newblock.go

```diff
@@ -71,11 +67,9 @@ func (node *Node) WaitForConsensusReadyv2(readySignal chan struct{}, stopChan ch
 					viewID := node.Consensus.GetViewID()
 					// add aggregated commit signatures from last block, except for the first two blocks
 
-					if node.NodeConfig.GetNetworkType() == nodeconfig.Mainnet {
-						if err = node.Worker.UpdateCurrent(coinbase); err != nil {
-							utils.GetLogger().Debug("Failed updating worker's state", "Error", err)
-							continue
-						}
+					if err = node.Worker.UpdateCurrent(coinbase); err != nil {
+						utils.GetLogger().Debug("Failed updating worker's state", "Error", err)
```

Comment:
use zero log please.

Reply:
I intentionally reverted that because zero log migration did not happen on r3 branch. or at least in these files changed in the pr